components/FiltersSection/index.tsx

```diff
@@ -1,71 +1,28 @@
+import { useFilterData } from '@/hooks/useFilterData';
 import { Button, Stack } from '@mui/material';
 import { Box } from '@mui/system';
 import { useRouter } from 'next/router';
-import { FC, FormEvent, useEffect, useState } from 'react';
+import { FC, FormEvent } from 'react';
 import { BaseButton } from '../StyledMuiComponents';
 import { Brands } from './brands';
 import { Categories } from './categories';
 import { Price } from './price';
 import { Rating } from './rating';
 import { FormSection } from './section';
 
-const initialFormData = {
-  categories: {} as { [key: string]: { [key: string]: boolean } },
-  brands: {} as { [key: string]: boolean },
-};
-
 type FiltersSection = {
```

Comment:
```suggestion
interface FiltersSectionProps {
```

`FiltersSection` is already defined here which is the component name